controllers/user.js

```diff
@@ -37,6 +37,9 @@ async function add(req, res) {
 
 
 async function verify(req, res) {
+  if (!req.query.token) {
+    throw new Error("Verification token is required.");
+  }
```

Comment:
👍 

```sh
$ curl http://localhost:6060/user/verify | grep -A1 "<h2>Error</h2>"

    <h2>Error</h2>
    <p> Verification token is required. </p>
```

Reply:
The error messages are friendlier, but not sure if we want to add UUID validation or anything, or if that is for a future PR...

Like, if I specify a bogus/typoed `?token=3` query param, it seems to do a DB query on the invalid UUID.v4.

```bash
$ curl https://fx-breach-alerts.herokuapp.com/user/verify\?token\=3 | grep -A1 "<h2>Error</h2>"

    <h2>Error</h2>
    <p> This email address is not subscribed to Firefox Monitor. </p>
```